teensy/main/main.ino

```diff
@@ -81,28 +93,41 @@ int count = 0;
 void print_all_sensors() {
   //prints a single string containing all sensor values
   //to be decoded by python script
-
+  String output = "";
```

Comment:
Can we implement some type of identifier for each line of data so that we know when exactly in time each data line is from? In the ESP, I use the internet connection to get the date and time. Maybe something like that here?